contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3073,6 +3077,35 @@ CREATE OR REPLACE FUNCTION sys.substring(string sys.NCHAR, i INTEGER, j INTEGER)
 RETURNS sys.NVARCHAR
 AS 'babelfishpg_tsql', 'tsql_varchar_substr' LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+CREATE OR REPLACE FUNCTION sys.substring(string sys.VARBINARY, i INTEGER, j INTEGER)
```

Comment:
Please use meaningful variable name instead of i and j everywhere.

Reply:
Done